http/src/main/resources/swagger/api-docs.yaml

```diff
@@ -2625,11 +2625,16 @@ components:
     AppStatus:
       type: string
       enum:
-        - DELETING
         - DELETED
+        - DELETING
         - ERROR
+        - PRECREATTING
```

Comment:
Nit, `PRECREATING`

Reply:
oh, we really shouldn't be exposing these statues..I'll have a separate PR to not exposing these details to users

Reply:
thanks.  Should I fix this spelling or remove all of the PRE ?

Reply:
I removed the PRE.  Now it's just adding STARTING and alphabetizing.